Backend/package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon your-app.js"
+    "start": "node your-app.js"
```

Comment:
You are doing right to keep the commits nice and small, so it's easy to trace and easier to review as well